templates/thank-you.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
```

Comment:
Did you discuss whether to use a dedicated page or a notification banner with @Sophie-32?